metricbeat/module/postgresql/postgresql.go

```diff
@@ -42,6 +42,7 @@ func init() {
 	}
 }
 
+//NewModule returns a new instance of the module
```

Comment:
Nit: we normally add a space ` ` after the `//`. Applies to few other places.

Thanks for adding the missing doc blocks.